public-lighting-demo-app/web-demo-app/src/main/resources/applicationContext.xml

```diff
@@ -1,33 +1,28 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <beans xmlns="http://www.springframework.org/schema/beans"
-      	xmlns:jpa="http://www.springframework.org/schema/data/jpa"
-      	xmlns:mvc="http://www.springframework.org/schema/mvc"
-      	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
-      	xsi:schemaLocation="
-		http://www.springframework.org/schema/beans
-		http://www.springframework.org/schema/beans/spring-beans-3.2.xsd
-		http://www.springframework.org/schema/data/jpa
-		http://www.springframework.org/schema/data/jpa/spring-jpa-1.2.xsd
-		http://www.springframework.org/schema/mvc
-		http://www.springframework.org/schema/mvc/spring-mvc-3.2.xsd
-		http://www.springframework.org/schema/tool
-        http://www.springframework.org/schema/tool/spring-tool-3.2.xsd">
-    <!--
-       Configures the location of static resources such as css files.
-       Requires Spring Framework 3.0 or higher.
-   -->
-    <mvc:resources mapping="/static/**" location="/static/"/>
+  xmlns:jpa="http://www.springframework.org/schema/data/jpa"
```

Comment:
Remove namespaces that are not needed, consider removing the versions (see also https://stackoverflow.com/questions/20894695/spring-configuration-xml-schema-with-or-without-version)